azurePipeline.yml

```diff
@@ -124,3 +126,26 @@ stages:
       - script: pip install -U mypy
       - script: mypy anonlink --ignore-missing-imports
         displayName: mypy
+
+- stage: publish
+  displayName: Publish packages to test feed
+  dependsOn: ['test']
+  jobs:
+  - job:
+    pool:
+      vmImage: 'ubuntu-latest'
+    variables:
+      # the name of an Azure artifacts feed to publish artifacts to
+      artifactFeed: anonlink
```

Comment:
Should we call it differently for the development feed in case we create another one open to the public?